grpc/send_incoming_call_notification.go

```diff
@@ -0,0 +1,51 @@
+package grpc
+
+import (
+	"context"
+	"encoding/binary"
+
+	pb "github.com/PretendoNetwork/grpc-go/friends"
+	nex "github.com/PretendoNetwork/nex-go"
+	friends_wiiu_types "github.com/PretendoNetwork/nex-protocols-go/friends-wiiu/types"
+	nintendo_notifications "github.com/PretendoNetwork/nex-protocols-go/nintendo-notifications"
+	nintendo_notifications_types "github.com/PretendoNetwork/nex-protocols-go/nintendo-notifications/types"
+	"github.com/PretendoNetwork/wiiu-chat-secure/globals"
+	"google.golang.org/grpc/metadata"
+)
+
+func SendIncomingCallNotification(caller uint32, target uint32) {
+	ctx := metadata.NewOutgoingContext(context.Background(), globals.GRPCFriendsCommonMetadata)
+
+	presence := friends_wiiu_types.NewNintendoPresenceV2()
+
+	presence.ChangedFlags = 0x1FF
+	presence.Online = true
+	presence.GameKey = friends_wiiu_types.NewGameKey()
+	presence.Unknown2 = 0x65
```

Comment:
I wonder if it's a coincidence that `0x65` is 101, which is the `RequestJoinGathering` notification category?

Reply:
Would make sense. I'm going to perform a test to see if sending `102` stops the ringing as well.

Reply:
Appears it does not correlate with the notification type, 102 does not stop ringing.

Reply:
Removing this *does* stop the ringing however, so I'm unsure if they're related.